src/Vatras/Translation/Lang/2CC-to-ADT.agda

```diff
@@ -25,8 +25,7 @@ open Eq.≡-Reasoning using (step-≡-⟨; step-≡-⟩; step-≡-∣; _∎)
 open IndexedSet using (_≅[_][_]_; ≅[]-sym; ≗→≅[])
 
 open import Vatras.Lang.All
-open 2CC using () renaming (2CC to 2CCSyntax) -- Necessary for disambiguation
-open 2CC using (2CC; 2CCL)
+open 2CC using (2CC; 2CCL; _-<_>-; _⟨_,_⟩)
```

Comment:
:raised_hands: Yes! Awesome! :)

Reply:
This is really great. This aligns the notation in the code base more with the paper (and other papers as well). :+1: